scripts/script.js

```diff
@@ -179,11 +179,12 @@ idApp.displayChoices = function(array, currentWindow, nextWindow) {
       userBio.generateBio(identityObject.name);
       const bioHeader = $('<h3>').text('Sample Personality')
       const shortBio = $('<p>').text(userBio.bioResults[0].shortBio)
-      const hiddenStuffs = $('<div>').attr('class', 'hiddenContents').append(bioHeader, shortBio)
+      const hiddenBio = $('<div>').attr('class', 'hiddenContents').append(bioHeader, shortBio)
 
-      const closeButton = $('<i>').attr('class', 'fas fa-plus')
+      const plusIcon = $('<i>').attr('class', 'fas fa-plus')
+      const closeButton = $('<button>').attr({class: 'closeButton', title: 'More Info', type: 'button'}).append(plusIcon);
```

Comment:
we should put an aria-label on this something like
.attr('aria-label', 'click this to show more information about this identity')

to make the button more accessible

Reply:
Agreed! Added the aria-labels in for the closeButton on windowB, and also for the labels on windowC. 

Weirdly, it was throwing me a syntax error when I tried to add it to the original group of attributes for closeButton on windowB (wouldn't let me add it in with class, title, and type). I chained it on as another individual .attr, and that worked. If you've got a 🔥 hot fix 🔥 for why it didn't let me group it in with the others, feel free!

Reply:
honestly I have no idea why that didn't work
the .attr() is super finnicky, I've written it the same way multiple times in the same file and sometimes it works and sometimes not. who knows.